CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## New Features:
 
+- Allow consecutive function triggers with `.then` and `.success` by [@aliabid94](https://github.com/aliabid94) in [PR 3430](https://github.com/gradio-app/gradio/pull/3430) 
```

Comment:
Would be good to include a code example in the changelog since this is a pretty big feature